content/en/database_monitoring/guide/connect_dbm_and_apm.md

```diff
@@ -12,18 +12,27 @@ private: true
 The features described on this page are in beta. Contact your Customer Success Manager to learn more about them.
 </div>
 
-This guide assumes that you have configured [Database Monitoring][1] and are using [APM][2].
+This guide assumes that you have configured [Database Monitoring][1] and are using [APM][2]. Connecting APM and DBM will inject APM trace identifiers into DBM data collection, which allows for correlation of these two data sources. This enables product features showing database information in the APM product, and APM data in the DBM product.  
+
 
 ## Before you begin
 
-Supported databases
-: postgres, mysql
+<dl>
+	<dt>Supported databases</dt>
+	<dd>postgres, mysql</dd>
+
+	<dt>Supported Agent versions</dt>
+	<dd>7.36.1+</dd>
```

Comment:
I think we should put the most recent agent version here? I think 7.36 is pretty old

Reply:
This is just the minimum required version, hence the `+`.

Reply:
yeah I get that, but often customers will just install this since its mentioned in the directions & then we have people running a bunch of really old agents

Reply:
This is also how we list agent versions for all our db integrations ([pg](https://docs.datadoghq.com/database_monitoring/setup_postgres/selfhosted?tab=postgres10), [mysql](https://docs.datadoghq.com/database_monitoring/setup_mysql/selfhosted/?tab=mysql56), [sqlserver](https://docs.datadoghq.com/database_monitoring/setup_sql_server/selfhosted/?tab=sqlserver2014)), so if we think that's an issue we should make a bunch of changes.
 
For this doc, I don't want to give customers the impression that they need to upgrade the agent to make linking work, since we are already asking customers to upgrade their tracers in the rest of the instructions.